lib/view_model/send/send_item.dart

```diff
@@ -0,0 +1,197 @@
+import 'package:cake_wallet/bitcoin/bitcoin_amount_format.dart';
+import 'package:cake_wallet/bitcoin/electrum_wallet.dart';
+import 'package:cake_wallet/entities/calculate_fiat_amount_raw.dart';
+import 'package:cake_wallet/entities/parse_address_from_domain.dart';
+import 'package:cake_wallet/entities/parsed_address.dart';
+import 'package:cake_wallet/monero/monero_amount_format.dart';
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+import 'package:mobx/mobx.dart';
+import 'package:cake_wallet/core/wallet_base.dart';
+import 'package:cake_wallet/monero/monero_wallet.dart';
+import 'package:cake_wallet/entities/calculate_fiat_amount.dart';
+import 'package:cake_wallet/entities/wallet_type.dart';
+import 'package:cake_wallet/store/dashboard/fiat_conversion_store.dart';
+import 'package:cake_wallet/store/settings_store.dart';
+import 'package:cake_wallet/generated/i18n.dart';
+
+part 'send_item.g.dart';
+
+const String cryptoNumberPattern = '0.0';
+
+class SendItem = SendItemBase with _$SendItem;
+
+abstract class SendItemBase with Store {
+  SendItemBase(this._wallet, this._settingsStore, this._fiatConversationStore)
+      :_cryptoNumberFormat = NumberFormat(cryptoNumberPattern) {
+    reset();
+    _setCryptoNumMaximumFractionDigits();
+    key = UniqueKey();
+  }
+
+  Key key;
+
+  @observable
+  String fiatAmount;
+
+  @observable
+  String cryptoAmount;
+
```

Comment:
if we have access to a wallet in this class, why we cannot to calculate result amount (crypto amount or "sendAll" amount) here ?